services/config.go

```diff
@@ -59,3 +61,18 @@ func (cfg Configuration) NewOptions() Options {
 	}
 	return opts
 }
+
+// ElectionConfiguration is for configuring election timeouts and TTLs
+type ElectionConfiguration struct {
+	LeaderTimeout time.Duration `yaml:"leaderTimeout"`
```

Comment:
Can you make them all optional fields? like [this](https://github.com/m3db/m3msg/blob/master/producer/config/buffer.go)